### ISSUE TITLE
Chip Aliases do not sort properly

### ISSUE THREAD

**Issue by mggates39** [mggates39/ChipInventory#2]
When searching for chip definitions, sometimes there are SN aliases before the numbered ones.

**Comment by mggates39** [comment]
Turns out, the alias entry code was leaving leading spaces.  This was corrected in code.  Execute the following SQL to clean up your existing data.
```
update aliases set alias_chip_number = trim(alias_chip_number)
where alias_chip_number like ' %' OR  alias_chip_number like '% ';
commit;
```
`

**Comment by mggates39** [comment]
Fixed by Pull Request #3
